### ISSUE TITLE
Help wanted: Updates to remove tensorflow warnings

### ISSUE THREAD

**Issue by blue-fish** [CorentinJ/Real-Time-Voice-Cloning#444]
Would someone please submit a pull request that replaces outdated Tensorflow function calls in the toolbox to avoid the warnings that show up when starting the toolbox, or running demo_cli.py?

For example, this is what we want to fix:
```
WARNING:tensorflow:From Real-Time-Voice-Cloning/synthesizer/models/modules.py:113: LSTMCell.__init__ (from tensorflow.python.ops.rnn_cell_impl) is deprecated and will be removed in a future version.
Instructions for updating:
This class is equivalent as tf.keras.layers.LSTMCell, and will be replaced by that in Tensorflow 2.0.
WARNING:tensorflow:From Real-Time-Voice-Cloning/synthesizer/models/modules.py:420: conv1d (from tensorflow.python.layers.convolutional) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.keras.layers.Conv1D` instead.
```

**Comment by mbdash** [comment]
I can't promise anything since I have never used GitHub for anything else then cloning.
But I could try over the weekend if no one else did it by then.

**Comment by blue-fish** [comment]
With #472 we should not bother with this as I expect we will remove the tensorflow-based synthesizer from the repo. When that is merged, this issue will be closed along with #370.

**Comment by blue-fish** [comment]
Actually I will go ahead and close this one now, and reopen it if it is needed.
